public/admin/config.yml

```diff
@@ -673,5 +673,18 @@ collections:
         widget: "string"
         required: false
         hint: "Luma Calendar URL for the event. Include relevant tags when appropriate, e.g., https://lu.ma/filecoin-events?tag=fil%20bangkok%202024"
+      - name: luma-events-section
+        label: "Luma Events Section"
+        widget: "object"
+        required: false
+        fields:
+          - name: "title"
+            label: "Section Title"
+            widget: "string"
+            required: false
+          - name: "embed-link"
```

Comment:
Maybe a hint "Paste the embed URL for your Luma event here. This will allow the events to be displayed directly on the page."